deepchecks/vision/metrics_utils/evalution_functions.py

```diff
@@ -0,0 +1,63 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module for evalution functions."""
```

Comment:
how are evaluation functions different from metrics? Maybe call it confusion_matrix_counts_metrics?